website/docs/terms/dataframe.md

```diff
@@ -0,0 +1,24 @@
+---
+id: dataframe
+title: DataFrame
+displayText: dataframe  
+hoverSnippet: A DataFrame is a two-dimensional data structure (rows and columns). It's the most common way of representing and interacting with large datasets in Python.
+---
+:::important This page could use some love
+This term would benefit from additional depth and examples. Have knowledge to contribute? [Create a discussion in the docs.getdbt.com GitHub repository](https://github.com/dbt-labs/docs.getdbt.com/discussions) to begin the process of becoming a glossary contributor!
+:::
+
+<!--- Useful reference: https://databricks.com/glossary/what-are-dataframes --->
+
+A DataFrame is a way of storing and manipulating tabular data in Python. (It's also used in other languages popular for data processing, such as R and Scala.)
+
+It's possible to string together a number of DataFrame transformations. For example, if `df` represents a DataFrame containing one row per person living in the Eastern United States over the last decade, we can calculate the number of people living in Philly each year:
```

Comment:
consider using `Philadelphia` unstead of `Philly` to be less US centric, less colloquial

Reply:
```suggestion
It's possible to string together a number of DataFrame transformations. For example, if `df` represents a DataFrame containing one row per person living in the Eastern United States over the last decade, we can calculate the number of people living in Philadelphia each year:
```